R/get_ds_rt.R

```diff
@@ -22,34 +23,46 @@
 #'  * `"zero"` sets specific rates evaluated to `NA` as 0. In addition, when
-#'  `std_popn` contains 0, the respective stratum is excluded from the calculation of
-#'  the directly standardized rate. However, when `std_popn` contains `NA`, the
-#'  directly standardized rate is evaluated to `NA`. When `output_type = "counts"`,
-#'  the expected counts are calculated *after* specific rates of `NA` have been set to 0.
-#'  *Confidence intervals cannot be constructed when this argument is used.*
+#'  `std_popn` contains 0, the respective stratum is excluded from the
+#'  calculation of the directly standardized rate. However, when `std_popn`
+#'  contains `NA`, the directly standardized rate is evaluated to `NA`. When
+#'  `output_type = "counts"`, the expected counts are calculated *after*
+#'  specific rates of `NA` have been set to 0. *Confidence intervals cannot be
+#'  constructed when this argument is used.*
 #' @param dist A string to indicate the type of probability distribution for the
-#' confidence interval to follow. Can be either `"normal"` or `"log normal"`.
-#' This parameter is case insensitive.
+#' confidence interval to follow. Can be either `"normal"`, `"log normal"`, or
+#' `"gamma"`. This parameter is case insensitive.
 #' @param interval A scalar, between 0 and 1, indicating the width of the
-#' confidence interval. For example, for a 95% confidence interval, use `interval = 0.95`.
+#' confidence interval. For example, for a 95% confidence interval, use
+#' `interval = 0.95`.
+#' @param method A string specifying the method to use when using gamma
+#' distribution. Use `"tcz06"` (default) for the method proposed by Tiwari,
+#' Clegg, and Zou (2006) and `"ff97"` for the more conservative method proposed
+#' by Fay and Feuer (1997). This parameter must be left empty when calculating
+#' confidence intervals using other distributions.
 #'
 #' @details This low-level function assumes that the counts in each stratum,
 #' the size of each stratum in the observed population and the size of each
 #' stratum in the standard population are provided in the same order.
 #'
 #' To construct confidence intervals for the rate estimates, arguments must be
 #' supplied to both `dist` and `interval`. In general, *the normal distribution
-#' should not be used to construct confidence intervals when the standard deviation
-#' is greater than a third of the mean*. The function throws a warning if this is the case.
+#' should not be used to construct confidence intervals when the standard
+#' deviation is greater than a third of the mean*. A warning is raised if this
+#' is the case.
 #'
 #' @return
-#' If `output_type = "rate"` (default) and neither `dist` or `interval` is supplied,
-#' the directly standardized rate is returned as a numeric vector.
+#' If `output_type = "rate"` (default) and neither `dist` or `interval` is
+#' supplied, the directly standardized rate is returned as a numeric vector.
```

Comment:
I think "neither `dist` or `interval` is" should read "neither `dist` **n**or `interval` is"

Reply:
Done